app/routes/api.js

```diff
@@ -46,12 +46,13 @@ exports.bus = function(req, res){
 exports.restaurant = function(req, res){
    
     var routing = apiUrl + "restaurant";
-    console.log(routing);
+    console.log(routing);    
+    let questionPick = getRandomInt(3);
```

Comment:
API currently only has one question for restaurant, so before this works we need to make a change to the API.

Reply:
all battlegrounds have been updated to 3 questions each